test/e2e/storage/vsphere/vsphere_volume_datastore.go

```diff
@@ -67,17 +71,33 @@ var _ = utils.SIGDescribe("Volume Provisioning on Datastore [Feature:vsphere]",
 		By("Invoking Test for invalid datastore")
 		scParameters[Datastore] = InvalidDatastore
 		scParameters[DiskFormat] = ThinDisk
-		err := invokeInvalidDatastoreTestNeg(client, namespace, scParameters)
+		err := invokeDatastoreTestNeg(client, namespace, scParameters)
 		Expect(err).To(HaveOccurred())
 		errorMsg := `Failed to provision volume with StorageClass \"` + DatastoreSCName + `\": The specified datastore ` + InvalidDatastore + ` is not a shared datastore across node VMs`
 		if !strings.Contains(err.Error(), errorMsg) {
 			Expect(err).NotTo(HaveOccurred(), errorMsg)
 		}
 	})
+
+	It("verify dynamically provisioned pv using storageclass fails on a local datastore", func() {
+		By("Invoking Test for local datastore")
+		localDatastore = os.Getenv("LOCAL_DATASTORE")
+		if localDatastore == "" {
+			framework.Skipf("Environment variable LOCAL_DATASTORE is not set. Skipping the test.")
+		}
+		scParameters[Datastore] = localDatastore
```

Comment:
I would recommend printing value of localDatastore variable. A wrongly set environment variable would go unnoticed otherwise.

Reply:
Add a log message for this.